templates/whatson/event_page.html

```diff
@@ -9,154 +9,156 @@
 {% endblock %}
 
 {% block content %}
-    {% include "includes/event_hero.html" with title=page.title image=page.lead_image event_type=page.event_type access_descriptor=page.primary_access_type %}
-
-    <div class="event-details-banner">
-        <div class="tna-container tna-container--align-center tna-container--space-between">
-            <div class="tna-column tna-column--width-3-4 tna-column--container">
-                <div class="event-meta event-meta--inline">
-                    <ul class="event-meta__list event-meta__list">
-                        {% if page.start_date %}
-                            <li class="event-meta__item">
-                                <i class="fa-regular fa-calendar event-meta__icon"></i>
-
-                                <div class="event-meta__details">
-                                    <p>{{ page.start_date|date:"l j F, H:i" }}</p>
-                                </div>
-                            </li>
-                        {% endif %}
-                        {% if page.price_range %}
+    <article>
+        {% include "includes/event_hero.html" with title=page.title image=page.lead_image event_type=page.event_type access_descriptor=page.primary_access_type %}
+
+        <div class="event-details-banner">
+            <div class="tna-container tna-container--align-center tna-!--padding-vertical-s">
```

Comment:
I don't think `!` is valid in a  class name

Reply:
I can see we're escaping the ! in the `overrides.scss` file but would be good to understand the reasoning behind this @ahosgood

Reply:
Using an "!" in a class name is fine - I've copied this approach from GOV.UK Frontend.

`tna-!--` classes are ones that have `!important` rules so we know just by looking that the padding is defined by this class and we shouldn't need to overwrite it. If we did need to change the bottom padding (for example) then we'd change to using the `tna-!--padding-top-s` class and explicitly define the padding for the bottom.